Example/AuthViewController.swift

```diff
@@ -56,8 +57,12 @@ final class AuthViewController: UIViewController {
         }
     }
     
-    func presentError(_ error: Error) {
-        self.showMessage("Error: \(error.localizedDescription)")
+    func showError(_ error: Error) {
```

Comment:
I suggest renaming this to `showMessage` as well, except accept an Error type instead of string. Let Swift figure out which method to invoke, based on the type of the argument.

Reply:
Done.